audiorecorder2-dev/audiorecorder2/audiorecorder2.rb

```diff
@@ -73,34 +74,124 @@ def EmbedBEXT(targetfile)
       end
     end
 
+    def PostRecord(targetfile)
+      window(title: "Post-Record Options", width: 600, height: 500) do
+        trimcheck = nil
+        @pretrim = $outputdir + '/' + File.basename(targetfile, File.extname(targetfile)) + '_untrimmed' + '.wav'
+        @finaloutput = targetfile
+        para targetfile
+        waveform = image("AUDIORECORDERTEMP.png")
+        @start_trim = nil
+        @end_trim_length = nil
+
+        def SetUpTrim(input)
```

Comment:
reason for declaring a method inside of a method?

Reply:
cuz it seemed to work within the context of shoes? Like due to the flow of the digitization process the PostRecord method is also the Post Record window, and it seemed to make intellectual sense to group the SetUpTrim within that 'window' which is also a method? hahaha

Reply:
Ohhh. Yes, I haven't used shoes!

Reply:
I mean, I'm winging it here! I don't know if this was a good reason or not hahaha